README.md

```diff
@@ -130,12 +132,14 @@ parameters.
 Configuration
 -------------
 
-* `BACKUP_K8S_CRONJOB_HISTORYLIMIT_FAILURE` (default `1`)
-* `BACKUP_K8S_CRONJOB_HISTORYLIMIT_SUCCESS` (default `3`)
-* `BACKUP_K8S_CRONJOB_BACKUP_SCHEDULE` (default `"0 0 * * *"`, once a day at 
-  midnight)
-* `BACKUP_K8S_CRONJOB_RESTORE_SCHEDULE` (default `None`, periodic restore is 
-  disabled)
+* `BACKUP_K8S_CRONJOB_HISTORYLIMIT_FAILURE` (default: `1`)
+* `BACKUP_K8S_CRONJOB_HISTORYLIMIT_SUCCESS` (default: `3`)
+* `BACKUP_K8S_CRONJOB_STARTING_DEADLINE_SECONDS` (default: `60`)
+* `BACKUP_K8S_CRONJOB_BACKUP_ENABLE` (default: `True`, periodic backup is enabled.)
```

Comment:
I think using the lowercase variant (`true`) is the better option here. Even though you set the default to the Python literal `True`, PyYAML renders any such boolean as `true` when Tutor writes back the config file. Thus, if a user sets this to `True` with a capital T, then `tutor config save` will just rewrite it as `true`, so we might as well use the lowercase version in the README from the get-go.

Reply:
Makes sense.